src/main/webapp/mentors.js

```diff
@@ -11,6 +11,56 @@ function initForm() {
   return;
 }
 
+/**
+ * A Mentor
+ * @typedef {Object} Mentor
+ * @property {string List} interestTags - The user's tags
+ * @property {string List} projectIDs - The user's projects
```

Comment:
nit: This might not be [valid JSDoc](https://jsdoc.app/tags-type.html).

```suggestion
 * @property {string[]} interestTags - The user's tags
 * @property {string[]} projectIDs - The user's projects
```